CMakeLists.txt

```diff
@@ -59,7 +59,10 @@ function(message)
 endfunction()
 
 option(PROFILE_TESTS "Profile tests" OFF)
-set(PYTHON python3)
+
+# Add CCF_PYTHON_PRELAUNCH=unbuffer to env when calling cmake to get coloured
+# test output
+set(PYTHON "$ENV{CCF_PYTHON_PRELAUNCH}" python3)
```

Comment:
Needs defining in CI yamls to get it to work?

Reply:
I think it's fine if we don't turn this on in the CI. The coloured logs show in some views, but appear as escape-strings in others making it really hard to read/copy.

Reply:
There's a $CI we can use though, to have it on locally by default?